blueman/gui/GenericList.py

```diff
@@ -127,11 +141,11 @@ def get(self, iterid, *items):
                     if items[i] in self.ids:
                         ret[items[i]] = self.liststore.get(tree_iter, self.ids[items[i]])[0]
         else:
-            return False
+            return {}
 
         return ret
 
-    def get_iter(self, path):
+    def get_iter(self, path: Union[Gtk.TreePath, int, str]) -> Optional[Gtk.TreeIter]:
         if path is None:
```

Comment:
Path may be None so the Union is optional, right? If not then the check for None can be removed.

Reply:
You're right. At least `ManagerDeviceList.level_setup_event` might theoretically pass `None`, however `mypy` does not know that due to no GObject stubs as it sees just `Any`. Also `mypy --strict-equality` would catch this but it sees `path` as `Any` as well, as `Gtk.TreePath` is `Any`.